pkg/snapshot/snapshot.job.template.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: snapshot
+spec:
+  template:
+    spec:
+      containers:
+        - name: tyk-operator
+          image: tykio/tyk-operator:v0.13
```

Comment:
Could the operator version be made dynamic?

Reply:
yeah for sure.